packages/ui/src/components/Transactions/Transaction.tsx

```diff
@@ -129,8 +129,7 @@ export default styled(Transaction)(
   }
 
   .callIcon {
-    font-size: 7rem;
```

Comment:
Here, the font size is the issue, if I put back this line, I get what I expect
![image](https://github.com/ChainSafe/Multix/assets/33178835/17dfa805-b212-40c2-9bf3-7a848ed949b5)